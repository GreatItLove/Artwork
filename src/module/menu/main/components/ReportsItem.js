import React from 'react';
import { ListItem } from 'material-ui/List';
import { NavLink } from 'react-router-dom';
import PropTypes from 'prop-types';

import { REPORTS_PATH } from '../../../../config/routesPatch';

const propTypes = {
  style: PropTypes.object.isRequired,
  styleChild: PropTypes.object.isRequired
};

/**
 * @function ReportsItem
 * @param {Object} style
 * @param {Object} styleChild
 * @returns {XML}
 */
function ReportsItem({ style, styleChild }) {
  return (
    <ListItem
      key={REPORTS_PATH}
      primaryText="Reports"
      innerDivStyle={style}
      leftIcon={<i className="material-icons">insert_chart</i>}
      primaryTogglesNestedList={true}
      nestedItems={[
        <ListItem
          key="DataLibrary"
          innerDivStyle={styleChild}
          primaryText="Data Library"
          containerElement={<NavLink to={REPORTS_PATH} exact />}
        />,
        <ListItem key="QueryBuilder" innerDivStyle={styleChild} primaryText="Query Builder" />,
        <ListItem key="Exports" innerDivStyle={styleChild} primaryText="Exports" />,
        <ListItem
          key="Artists"
          innerDivStyle={styleChild}
          primaryText="Artists"
          primaryTogglesNestedList={true}
          nestedItems={[
            <ListItem key="ArtistsInfo" innerDivStyle={styleChild} primaryText="Artists Info" />,
            <ListItem key="Exports" innerDivStyle={styleChild} primaryText="Exports" />,
            <ListItem key="ArtistsSEO" innerDivStyle={styleChild} primaryText="Artists SEO" />
          ]}
        />,
        <ListItem
          key="Artwork"
          innerDivStyle={styleChild}
          primaryText="Artwork"
          primaryTogglesNestedList={true}
          nestedItems={[
            <ListItem key="Insurance" innerDivStyle={styleChild} primaryText="Insurance" />,
            <ListItem key="COA" innerDivStyle={styleChild} primaryText="COA" />,
            <ListItem key="ConsignedOut" innerDivStyle={styleChild} primaryText="Consigned Out" />,
            <ListItem key="ArtworkDescriptions" innerDivStyle={styleChild} primaryText="Artwork Descriptions" />,
            <ListItem key="ArtworkImageNames" innerDivStyle={styleChild} primaryText="Artwork Image Names" />,
            <ListItem key="PrintLabelsLists" innerDivStyle={styleChild} primaryText="Print Labels & Lists" />
          ]}
        />,
        <ListItem
          key="Contacts"
          innerDivStyle={styleChild}
          primaryText="Contacts"
          primaryTogglesNestedList={true}
          nestedItems={[
            <ListItem key="AllContacts" innerDivStyle={styleChild} primaryText="All Contacts" />,
            <ListItem key="ContactMailingList" innerDivStyle={styleChild} primaryText="Contact Mailing List" />,
            <ListItem key="EmailList" innerDivStyle={styleChild} primaryText="Email List" />,
            <ListItem key="MarketingInfo" innerDivStyle={styleChild} primaryText="Marketing Info" />,
            <ListItem key="AllBuyers" innerDivStyle={styleChild} primaryText="All Buyers" />,
            <ListItem key="GrossSpent" innerDivStyle={styleChild} primaryText="Gross Spent" />
          ]}
        />,
        <ListItem
          key="Invoices"
          innerDivStyle={styleChild}
          primaryText="Invoices"
          primaryTogglesNestedList={true}
          nestedItems={[
            <ListItem key="MostRecentInvoices" innerDivStyle={styleChild} primaryText="Most Recent Invoices" />,
            <ListItem
              key="ItemsLocationsSalespersons"
              innerDivStyle={styleChild}
              primaryText="Items-Locations/Salespersons"
            />,
            <ListItem key="ItemsArtistConsignors" innerDivStyle={styleChild} primaryText="Items-Artist/Consignors" />,
            <ListItem key="CustomersCountry" innerDivStyle={styleChild} primaryText="Customers - Country" />,
            <ListItem key="AllInvoices" innerDivStyle={styleChild} primaryText="All Invoices" />
          ]}
        />,
        <ListItem
          key="Consignments"
          innerDivStyle={styleChild}
          primaryText="Consignments"
          primaryTogglesNestedList={true}
          nestedItems={[
            <ListItem key="ConsignorContactInfo" innerDivStyle={styleChild} primaryText="Consignor Contact Info" />,
            <ListItem key="ExportConsignorsXLS" innerDivStyle={styleChild} primaryText="Export Consignors (XLS)" />,
            <ListItem key="ExportCommissionsXLS" innerDivStyle={styleChild} primaryText="Export Commissions (XLS)" />
          ]}
        />
      ]}
    />
  );
}

ReportsItem.propTypes = propTypes;

export default ReportsItem;
