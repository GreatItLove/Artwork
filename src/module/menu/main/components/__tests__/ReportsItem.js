import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import ReportsItem from '../ReportsItem';

describe('ReportsItem', () => {
  it('Snapshot of ReportsItem', () => {
    const wrapper = shallow(<ReportsItem style={{}} styleChild={{}} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
