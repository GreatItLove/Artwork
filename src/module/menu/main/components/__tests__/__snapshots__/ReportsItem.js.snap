// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReportsItem Snapshot of ReportsItem 1`] = `
<ListItem
  autoGenerateNestedIndicator={true}
  containerElement="span"
  disableKeyboardFocus={false}
  disabled={false}
  initiallyOpen={false}
  innerDivStyle={Object {}}
  insetChildren={false}
  leftIcon={
    <i
      className="material-icons"
    >
      insert_chart
    </i>
  }
  nestedItems={
    Array [
      <ListItem
        autoGenerateNestedIndicator={true}
        containerElement={
            <NavLink
                activeClassName="active"
                ariaCurrent="true"
                exact={true}
                to="/reports"
            />
        }
        disableKeyboardFocus={false}
        disabled={false}
        initiallyOpen={false}
        innerDivStyle={Object {}}
        insetChildren={false}
        nestedItems={Array []}
        nestedLevel={0}
        onKeyboardFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onNestedListToggle={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        open={null}
        primaryText="Data Library"
        primaryTogglesNestedList={false}
        secondaryTextLines={1}
    />,
      <ListItem
        autoGenerateNestedIndicator={true}
        containerElement="span"
        disableKeyboardFocus={false}
        disabled={false}
        initiallyOpen={false}
        innerDivStyle={Object {}}
        insetChildren={false}
        nestedItems={Array []}
        nestedLevel={0}
        onKeyboardFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onNestedListToggle={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        open={null}
        primaryText="Query Builder"
        primaryTogglesNestedList={false}
        secondaryTextLines={1}
    />,
      <ListItem
        autoGenerateNestedIndicator={true}
        containerElement="span"
        disableKeyboardFocus={false}
        disabled={false}
        initiallyOpen={false}
        innerDivStyle={Object {}}
        insetChildren={false}
        nestedItems={Array []}
        nestedLevel={0}
        onKeyboardFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onNestedListToggle={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        open={null}
        primaryText="Exports"
        primaryTogglesNestedList={false}
        secondaryTextLines={1}
    />,
      <ListItem
        autoGenerateNestedIndicator={true}
        containerElement="span"
        disableKeyboardFocus={false}
        disabled={false}
        initiallyOpen={false}
        innerDivStyle={Object {}}
        insetChildren={false}
        nestedItems={
            Array [
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Artists Info"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Exports"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Artists SEO"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
              ]
        }
        nestedLevel={0}
        onKeyboardFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onNestedListToggle={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        open={null}
        primaryText="Artists"
        primaryTogglesNestedList={true}
        secondaryTextLines={1}
    />,
      <ListItem
        autoGenerateNestedIndicator={true}
        containerElement="span"
        disableKeyboardFocus={false}
        disabled={false}
        initiallyOpen={false}
        innerDivStyle={Object {}}
        insetChildren={false}
        nestedItems={
            Array [
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Insurance"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="COA"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Consigned Out"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Artwork Descriptions"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Artwork Image Names"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Print Labels & Lists"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
              ]
        }
        nestedLevel={0}
        onKeyboardFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onNestedListToggle={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        open={null}
        primaryText="Artwork"
        primaryTogglesNestedList={true}
        secondaryTextLines={1}
    />,
      <ListItem
        autoGenerateNestedIndicator={true}
        containerElement="span"
        disableKeyboardFocus={false}
        disabled={false}
        initiallyOpen={false}
        innerDivStyle={Object {}}
        insetChildren={false}
        nestedItems={
            Array [
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="All Contacts"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Contact Mailing List"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Email List"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Marketing Info"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="All Buyers"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Gross Spent"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
              ]
        }
        nestedLevel={0}
        onKeyboardFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onNestedListToggle={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        open={null}
        primaryText="Contacts"
        primaryTogglesNestedList={true}
        secondaryTextLines={1}
    />,
      <ListItem
        autoGenerateNestedIndicator={true}
        containerElement="span"
        disableKeyboardFocus={false}
        disabled={false}
        initiallyOpen={false}
        innerDivStyle={Object {}}
        insetChildren={false}
        nestedItems={
            Array [
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Most Recent Invoices"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Items-Locations/Salespersons"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Items-Artist/Consignors"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Customers - Country"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="All Invoices"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
              ]
        }
        nestedLevel={0}
        onKeyboardFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onNestedListToggle={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        open={null}
        primaryText="Invoices"
        primaryTogglesNestedList={true}
        secondaryTextLines={1}
    />,
      <ListItem
        autoGenerateNestedIndicator={true}
        containerElement="span"
        disableKeyboardFocus={false}
        disabled={false}
        initiallyOpen={false}
        innerDivStyle={Object {}}
        insetChildren={false}
        nestedItems={
            Array [
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Consignor Contact Info"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Export Consignors (XLS)"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Export Commissions (XLS)"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
              ]
        }
        nestedLevel={0}
        onKeyboardFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onNestedListToggle={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        open={null}
        primaryText="Consignments"
        primaryTogglesNestedList={true}
        secondaryTextLines={1}
    />,
    ]
  }
  nestedLevel={0}
  onKeyboardFocus={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onNestedListToggle={[Function]}
  onTouchEnd={[Function]}
  onTouchStart={[Function]}
  open={null}
  primaryText="Reports"
  primaryTogglesNestedList={true}
  secondaryTextLines={1}
/>
`;
